src/contract_client/utils.ts

```diff
@@ -1,7 +1,11 @@
+// @ts-ignore
+import * as jsxdr from "@stellar/js-xdr";
+import { xdr } from "@stellar/stellar-base";
+
 /**
  * The default timeout for waiting for a transaction to be included in a block.
  */
-export const DEFAULT_TIMEOUT = 5 * 60;
+export const DEFAULT_TIMEOUT = 10;
```

Comment:
Intentionally undoing? Looks like a bad upstream sync/merge.

Reply:
Good catch, sorry about that!